test/test_images/probe_helper/probe_helper_test.go

```diff
@@ -257,21 +216,32 @@ func testPubsubClient(ctx context.Context, t *testing.T, projectID string) (*pub
 	return c, close
 }
 
+func testStorageClient(ctx context.Context, t *testing.T) (*storage.Client, chan *http.Request, func()) {
+	gotRequest := make(chan *http.Request, 1)
+	srv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		// The test Cloud Storage server forwards the client's generated HTTP requests.
+		body, err := ioutil.ReadAll(r.Body)
+		if err != nil {
+			logging.FromContext(ctx).Fatal("Test Cloud Storage server could not read request body.")
+		}
+		r.Body = ioutil.NopCloser(bytes.NewBuffer(body))
+		gotRequest <- r
+		w.Write([]byte("{}"))
+	}))
+	c, err := storage.NewClient(ctx, option.WithHTTPClient(srv.Client()), option.WithEndpoint(srv.URL))
+	if err != nil {
+		t.Fatalf("Failed to create test storage client: %v", err)
+	}
+	return c, gotRequest, srv.Close
+}
+
 type eventAndResult struct {
 	event      *cloudevents.Event
 	wantResult protocol.Result
 }
 
-type requestData struct {
-	method string
-	url    string
-	body   string
-}
-
 func TestProbeHelper(t *testing.T) {
-	t.Skip("Skip this test from running on Prow as it is only for local development.")
-
-	os.Setenv("K_SINK", dummyBrokerURL)
+	os.Setenv("K_SINK", testBrokerURL)
```

Comment:
We should either use t.Skip() or fix the flakiness of this test. For now we can continue using t.Skip.

Reply:
Done.